msbuild/Messaging/Xamarin.Messaging.Build/Handlers/VerifyXcodeVersionMessageHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using Xamarin.Messaging.Client;
+using System.Threading.Tasks;
+using Xamarin.MacDev;
+using Xamarin.Messaging.Build.Contracts;
+
+namespace Xamarin.Messaging.Build {
+	public class VerifyXcodeVersionMessageHandler : RequestHandler<VerifyXcodeVersionMessage, VerifyXcodeVersionResult> {
+		protected override Task<VerifyXcodeVersionResult> ExecuteAsync (VerifyXcodeVersionMessage message)
+		{
+			var result = new VerifyXcodeVersionResult ();
+
+			try {
+				var xcodeVersion = AppleSdkSettings.XcodeVersion;
+				var recommendedXcodeVersion = GetRecommendedXcodeVersion ();
+
+				result.IsCompatibleVersion = xcodeVersion >= recommendedXcodeVersion;
+				result.XcodeVersion = xcodeVersion.ToString();
+				result.RecommendedXcodeVersion = recommendedXcodeVersion.ToString();
+			} catch {
+				result.IsCompatibleVersion = false;
+			}
+
+			return Task.FromResult (result);
+		}
+
+		Version GetRecommendedXcodeVersion ()
+		{
+			var monoTouchSdk = GetMonoTouchSdk ();
+
+			return monoTouchSdk == null ? new Version (0, 0) : monoTouchSdk?.RecommendedXcodeVersion;
+		}
+
+		MonoTouchSdk GetMonoTouchSdk ()
+		{
+			var sdk = default (MonoTouchSdk);
+
+			foreach (var sdkDir in MonoTouchSdk.DefaultLocations) {
+				if (MonoTouchSdk.ValidateSdkLocation (sdkDir)) {
+					sdk = new MonoTouchSdk (sdkDir);
+					break;
+				}
+			}
+
+			return sdk;
```

Comment:
Couldn't this be simplified to:

```suggestion
			return Sdks.XamIOS;
```

Ref:
https://github.com/xamarin/xamarin-macios/blob/f01fde5cd9a7ffffcdc8d241200c35988700fa00/msbuild/Xamarin.MacDev.Tasks.Core/Sdks.cs#L16